CONTRIBUTING.md

```diff
@@ -0,0 +1,80 @@
+
+## Want to contribute as player?
+We appreciate feedback from our players as it is you why we are doing all of this. Without you, we would not be where we are.
+
+### Found a bug?
+* **Ensure the bug was not already reported** by searching on GitHub under [Issues](https://github.com/GregTechCE/GregTech/issues).
+* If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/GregTechCE/GregTech/issues/new/choose) using **Bug report** template.
+Be sure to include a **title and clear description**, as much relevant information as possible, let the template be your guide.
+
+### Something is missing? Thought about a cool feature?
+* Check if there is similar idea under [Issues](https://github.com/GregTechCE/GregTech/issues) and if so but something is missing then please collaborate on it.
+* Nothing similar? Then [open a new issue](https://github.com/GregTechCE/GregTech/issues/new/choose) using **Feature request** template.
+
+### Have question regarding project?
+* We have Discord (link above), feel free to hang out and ask your questions. There are always some community members which may provide answers.
+* Discord not your thing? Okay, understandable, then please [open a new issue](https://github.com/GregTechCE/GregTech/issues/new/choose) using **Question** template.
+
+### Anything else?
+* Discord is right place for it.
+
+## Want to contribute some more?
+We are mostly organizing our self on Discord and it is preferred place to discuss issues or quickly exchange info.
+GitHub is more of input, output and storage kind of place.
+Of course if we made any decision regarding issues/PRs we will write it on GitHub, so it is available to anyone.
+
+### Know foreign language and want to contribute translation?
+* If we already have language files for it feel free to review them and let us know any mistakes or improvements.
+* We don't have then, and you feel brave enough to make them then please join us on Discord, so we can discuss it.
+
+### Don't know how to code?
+We will stay take your help. 
+* You can look through open issues with [status: unverified](https://github.com/GregTechCE/GregTech/labels/status%3A%20unverified) and confirm they are present.
+On simplest possible configuration and write down reproduction steps if they are missing. Or provide any other related information.
```

Comment:
"Please attempt to confirm them on the simplest possible configuration, and provide reproduction steps if the are missing from the original issue."

Reply:
I am not sure how to get this in can you please supply it via `Insert suggestion` feature.